js/components/frame.js

```diff
@@ -25,16 +25,14 @@ class Frame extends ImmutableComponent {
 
   updateWebview () {
     let src = this.props.frame.get('src')
-    const isAboutURL = isSourceAboutUrl(src)
-    if (isAboutURL) {
-      src = getTargetAboutUrl(src)
-    }
     let contentScripts = ['content/webviewPreload.js']
-    if (this.props.frame.get('location') === 'about:preferences') {
+    // NOTE: use src instead of location here because location is sometimes
+    // not updated when redirecting to about:certerror
+    if (src === 'about:preferences' || src === 'about:certerror') {
```

Comment:
Can't seem to get it to happen again. Will change it back to location instead of src.